persistence/sql/migrations/sql/20230614000001000000_hydra_login_challenge_format.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE selfservice_login_flows DROP COLUMN oauth2_login_challenge;
+ALTER TABLE selfservice_registration_flows DROP COLUMN oauth2_login_challenge;
+
+ALTER TABLE selfservice_login_flows ADD COLUMN oauth2_login_challenge_data TEXT NULL;
+ALTER TABLE selfservice_registration_flows ADD COLUMN oauth2_login_challenge_data TEXT NULL;
```

Comment:
You can make this an online migration by executing the ADD COLUMN only, and then later, in another release, execute DROP COLUMN.

Reply:
Nice, fixed :)